src/mono/wasm/runtime/dotnet-crypto-worker.js

```diff
@@ -101,6 +116,8 @@ var ChannelWorker = {
                 // Update the state
                 Atomics.store(this.comm, this.STATE_IDX, state);
 
+                this._release_lock();
```

Comment:
should this be released if there was any error thrown?

Reply:
What kind of error are you thinking?

Reply:
A bug in our code that causes an error to be thrown, for example. Do IIUC that the app would be stuck in that case as  the main thread would be spinning?

Reply:
> Do IIUC that the app would be stuck in that case as the main thread would be spinning?

Yeah, probably.

Reply:
I think a lot more work would be necessary to make this code "reentrant" on an error. It would probably be better tracked by https://github.com/dotnet/runtime/issues/69740, than in this PR.